index.bs

```diff
@@ -7914,10 +7904,25 @@ Constructors</h4>
 				[[mediacapture-streams#mediastream|MediaStream]] that has at least one
 				[[mediacapture-streams#mediastreamtrack|MediaStramTrack]] whose
 				<code>kind</code> attribute has the value <code>"audio"</code>,
-				throw an {{InvalidStateError}} and abort these steps.
-			3. Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
+				throw an {{InvalidStateError}} and abort these steps. Else, let
+				this stream be <var>inputStream</var>.
+			3. Let <var>tracks</var> the list of all
+				[[mediacapture-streams#mediastreamtrack|MediaStreamTrack]] of
+				<var>inputStream</var> that have a <code>kind</code> of
+				<code>"audio"</code>.
+			4. Sort the elements in <var>tracks</var> based on their <code>id</code>
+				attribute, ordering by Unicode code point
+			5. Let the first element of <var>tracks</var> be the
```

Comment:
Do we need to describe the rest of tracks as well?

Reply:
No we leave them alone, they are of no importance for us.

Reply:
ACK